bundles/AdminBundle/Controller/Admin/DataObject/ClassificationstoreController.php

```diff
@@ -1403,6 +1393,28 @@ protected function getConfigItem($config): array
         return $item;
     }
 
+    /**
+     * @param Classificationstore\KeyConfig $config
+     *
+     * @return array
+     */
+    protected function getKeyConfigItem($config): array
```

Comment:
Why not use type hint for param?

Reply:
Good point, added them for both method, since the class is internal and the methods are protected anyways